src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/LocalTests.vb

```diff
@@ -0,0 +1,68 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Symbols
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
```

Comment:
>Microsoft.CodeAnalysis.VisualBasic.UnitTests [](http://example.com/codeflow?start=10&length=44)

Microsoft.CodeAnalysis.VisualBasic.UnitTests.Symbols?

Reply:
The other files in this directory have also this namespace, e.g.:

https://github.com/dotnet/roslyn/blob/651269a5d647d78cfe2c836da501eed693df5b0b/src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/FieldTests.vb#L16